cmd/keymasterd/app.go

```diff
@@ -870,6 +870,7 @@ func (state *RuntimeState) reprocessUsername(username string) string {
 }
 
 const secretInjectorPath = "/admin/inject"
+const checkSealedPath = "/admin/checkSealed"
```

Comment:
instead of a check unsealed, better have an (is ready) endpoint? ... and maybe the endpoint return a json blob so that we can encode other data?

Reply:
If we go down this path, let's make it Kubernetes-compatible and return "200 OK" if unsealed and "404 Not Found" if sealed.

Reply:
yes please...

Reply:
Hm. Maybe the not ready case should be "503 not ready"?

Reply:
a 5XX makes more sense.

Reply:
Done.